x-pack/plugins/painless_lab/public/plugin.tsx

```diff
@@ -90,7 +91,15 @@ export class PainlessLabUIPlugin implements Plugin<void, void, PluginDependencie
         }
 
         const { renderApp } = await import('./application');
-        return renderApp(element, { I18nContext, http, uiSettings, links: getLinks(docLinks) });
+        const tearDownApp = renderApp(
+          element,
+          { I18nContext, http, uiSettings, links: getLinks(docLinks) },
+          { getIsNavDrawerLocked$ }
```

Comment:
Could we pass through the chrome dependency in the same object as http, uiSettings etc?